testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -1451,6 +1451,11 @@ protected void testReadMetadataWithRelationsConcurrentModifications(int readIter
         assertTrue(executor.awaitTermination(10, SECONDS));
     }
 
+    protected String getCreateTemplateForTableWithOneIntegerColumn()
```

Comment:
Add `@Language("SQL")` annotation and rename to `createTableSqlTemplateForConcurrentModifications()`.

Reply:
@ebyhr Thanks! Fixed :)